app/src/main/java/com/nervousfish/nervousfish/modules/pairing/AbstractPairingHandler.java

```diff
@@ -4,12 +4,12 @@
  * Contains common methods shared by all pairing modules.
  */
 @SuppressWarnings("PMD.AbstractClassWithoutAbstractMethod")
-abstract class APairingHandler {
+abstract class AbstractPairingHandler {
```

Comment:
Well, because the Checkstyle developers thought that that would be a good style...